### DISCUSSION TITLE
How to measure the width of a street with tram tracks in the car lanes?

### DISCUSSION THREAD

**Discussion by MichaelMarkert** [streetcomplete/StreetComplete#5203] [Q&A]
Hi to all, in my neighborhood is a broader street with two car lanes and two tram tracks (red in the picture) in between them (and at some points overlapping). So there are no curbs to use as a marker for the street width except at the sidewalk which would give only one width not two as requested. How should I proceed? Use the tracks itself as borders? Thanks in advance!

![Screenshot_20230818-162623~3](https://github.com/streetcomplete/StreetComplete/assets/101104547/d4f53531-0485-4548-a8b9-8e0371912aab)

**Comment by westnordost** [comment]
Hm, not sure... better ask in the openstreetmap forum. I'd probably say that in case the road is split up between left of and right of the tram line, the width measured for each the left and right side of the tram line should exclude the tram line. I.e. measure the width of the street from one curb to including the lane that is not the tram line lane.
